client/settings/action-creators.ts

```diff
@@ -39,7 +40,11 @@ export function mergeLocalSettings(
     try {
       await ipcRenderer.invoke('settingsLocalMerge', settings)
     } catch (err) {
-      dispatch(openSnackbar({ message: 'There was an issue saving the settings.' }))
+      dispatch(
+        openSnackbar({
+          message: i18n.t('settings.errors.merge', 'There was an issue saving the settings.'),
```

Comment:
nit: I feel like given the contents, this string should probably be called `settings.errors.save`